docs/content/mobile/ios/local-deploy.md

```diff
@@ -1,13 +1,28 @@
 [//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
 
-# Connecting to a local deployment
+# Connect to a local deployment
```

Comment:
best to revert those changes. @oblm is redoing them in his branch afaik

Reply:
Yup, I'll rebase on top of his changes once they have landed